spec/features/system/admin/instance_management_spec.rb

```diff
@@ -48,8 +48,10 @@
       end
 
       scenario 'I can see all instances' do
-        instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        create_list(:instance, 2)
+        last_page = Instance.page.total_pages
+        instances = Instance.order_by_id.page(last_page)
```

Comment:
If the order is by id, the 2 instances you created will be in the `instances` variable, so they are actually tested and checked.  Do you have any other idea for testing this ?